packages/shared/src/lib/featureManagement.ts

```diff
@@ -98,6 +98,32 @@ export class Features {
     'v5',
   ]);
 
+  static readonly CompanionPermissionPlacement = new Features(
+    'companion_permission_placement',
+    'off',
+    ['off', 'header', 'sidebar'],
+  );
+
+  static readonly CompanionPermissionTitle = new Features(
+    'companion_permission_title',
+    'The companion lets you comment and upvote directly on an article! 🤯',
+  );
+
+  static readonly CompanionPermissionDescription = new Features(
+    'companion_permission_description',
+    'Heads up! We need to ask for some extra permissions so you can enjoy the power of the companion.',
+  );
+
+  static readonly CompanionPermissionLink = new Features(
+    'companion_permission_link',
+    'Watch the Companion\nOverview',
```

Comment:
Not sure if we even need the new line?
Is it not just a design issue where they showcased the new line for that example?

Reply:
Yes, it is from the design. Not sure if the new line was intentional but it looked much better so I applied it.

Reply:
Ah ok perfect